src/Avalonia.Controls/Button.cs

```diff
@@ -351,6 +375,28 @@ private static void IsDefaultChanged(AvaloniaPropertyChangedEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Called when the <see cref="IsCancel"/> property changes.
+        /// </summary>
+        /// <param name="e">The event args.</param>
+        private static void IsCancelChanged(AvaloniaPropertyChangedEventArgs e)
```

Comment:
This code will add two separate handlers to the root if `IsCancel` and `IsDefault` are both set. Might the code be simpler if we just add a single handler if either of the properties is set, and then check which key is pressed in the handler?

Reply:
IMO, I do not like 2 events handled by the same handler and then this handler having "if" statement inside. I prefer 2 methods rather than "if".

Reply:
Hmm, I'm not convinced, but let's try it like this then 👍